src/Util/JsonLD.php

```diff
@@ -165,22 +165,22 @@ public static function compact($json)
 	}
 
 	/**
-	 * Fetches an element array from a JSON array
+	 * Fetches an element array from a JSON array, optionally filtering for a single sub-element by key
 	 *
-	 * @param $array
-	 * @param $element
-	 * @param $key
+	 * @param array $array
+	 * @param string $element
+	 * @param string $key
 	 *
 	 * @return array fetched element
 	 */
-	public static function fetchElementArray($array, $element, $key = '@id')
+	public static function fetchElementArray($array, $element, $key = null)
 	{
 		if (empty($array)) {
-			return null;
+			return [];
```

Comment:
We are testing for `null` here: https://github.com/friendica/friendica/blob/develop/src/Util/JsonLD.php#L232 and here: https://github.com/friendica/friendica/blob/develop/src/Util/JsonLD.php#L240 - that change won't work anymore.

Reply:
Ah, thanks.